gitpod-io/gitpod
# gitpod-cli: call segement directly for analytics

## Description
Removes a dependency on server to send analytics events from `gp` CLI. 

Supervisor is aware of the OwnerId and we expose it in WorkspaceInfo response.

<img width="200" alt="Screenshot 2023-01-17 at 23 05 34" src="https://user-images.githubusercontent.com/2318450/213081189-b1cbc4c0-c552-4761-99da-b095449e1761.png">

## Related Issue(s)

Fixes #

## How to test
1. Start a workspace https://af-gp-anal1fcb7cb30d.preview.gitpod-dev.com/workspaces
2. Run `gp rebuild` and generate some events
3. Check in Segment Staging Trusted and verify the event is there, including `UserId`

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
